README.md

```diff
@@ -402,22 +475,64 @@ Example `process_exec` and `process_exit` events can be:
 </p>
 </details>
 
-For the rest of the use cases we will use the Tetragon CLI to give the output.
+#### Use case 2: Privileged Execution
+
+Tetragon also provides the ability to check process capabilities and kernel namespaces.
+
+As a first step let's enable visibility to capability and namespace changes via the configmap
```

Comment:
It would be nice to describe what these flags do exactly in like 2 sentence. I think with this we expand every JSON event with caps/ns and we extract these when a process starts from a kernel struct. But I'm not 100% sure, would be nice to double check.